rebasehelper/spec_content.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
```

Comment:
You should also probably move its tests to a separate file

Reply:
Done. I've also added tests for Tags.